zeena-ahmad/style.css

```diff
@@ -0,0 +1,83 @@
+.sidebar {
+    display: inline-block;
+    height: 100%;
+    width: 220px;
+    background-color: #334856;
+    float: left;
+    padding-top: 100px;
+    padding-bottom:100%;
+   
+    
+  }
+  
+  aside a{
+    border: 1px solid white;
+    margin-left: auto;
+    color:white;
+    margin-right: 10px;
+    padding:5px 10px 5px 10px;
+    text-decoration:none;
+    font-size:8px;
+  }
+  
+  aside, h3{
+   color: white;
+   text-align: right;
+   padding-right:20px;
+  }
+  
+  aside p{
+    color:#668FAA;
+  }
+  #img{
+    width:40px;
+    height: 40px;
+    border-radius: 50%;
+    float:right;
+    margin-top: -20px;
+
+  }
+  .one{
+    margin:25px 100px 0px 300px;
+    font-family:courier;
+   
```

Comment:
After each section of the code try to put newline "empty line" like `Line:39`, `Line:44` ...etc
 and don't leave any empty lines in places like this `Line:43`, `Line:47`  in order to make your code clean